packages/tldraw/src/lib/ui/hooks/useClipboardEvents.ts

```diff
@@ -669,16 +675,27 @@ export function useNativeClipboardEvents() {
 			if (editor.user.getIsPasteAtCursorMode()) pasteAtCursor = !pasteAtCursor
 			if (pasteAtCursor) point = editor.inputs.currentPagePoint
 
-			// First try to use the clipboard data on the event
-			if (e.clipboardData && !editor.inputs.shiftKey) {
-				handlePasteFromEventClipboardData(editor, e.clipboardData, point)
-			} else {
-				// Or else use the clipboard API
-				navigator.clipboard.read().then((clipboardItems) => {
-					if (Array.isArray(clipboardItems) && clipboardItems[0] instanceof ClipboardItem) {
-						handlePasteFromClipboardApi(editor, clipboardItems, point)
+			const pasteFromEvent = () => {
+				if (e.clipboardData) {
```

Comment:
this isn't checking `shiftKey` anymore — shiftKey should do a "normal" / non-rich paste. is that being captured somewhere?

Reply:
This shift key toggles "paste at cursor" mode, but as far as I could tell was otherwise not actually doing anything here. I don't know why it was included in that check to be honest.

Reply:
ok, gotcha